little-sichuan/backend/src/services/product.service.ts

```diff
@@ -2,52 +2,44 @@ import { OutgoingMessage } from 'http';
 import { Double, getRepository } from 'typeorm';
 import { Product } from '../models';
 import { productRouter } from '../routes';
+import { HttpResponseModel } from '../regularmodule/HttpResponse.model';
+
+var Validator = require('jsonschema').Validator;
```

Comment:
try to use `import`, also do not validate in the services, do it in the routes controller